internal/metrics/metricsManager_test.go

```diff
@@ -0,0 +1,72 @@
+package metrics
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/google/uuid"
+	"github.com/openshift/assisted-service/models"
+	"github.com/prometheus/client_golang/prometheus"
+
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Metrcis Manager", func() {
+
+	const (
+		ocpVersion  = "4.6"
+		emailDomain = "redhat.com"
+	)
+
+	var (
+		mm        *MetricsManager
+		clusterID strfmt.UUID
+	)
+
+	BeforeEach(func() {
+		clusterID = strfmt.UUID(uuid.New().String())
+		prometheusRegistry := prometheus.DefaultRegisterer
+		mm = NewMetricsManager(prometheusRegistry)
+	})
+
+	AfterEach(func() {
+	})
+
+	Context("Validation metrics", func() {
+
+		It("Host validation metrics", func() {
+
+			tests := []struct {
```

Comment:
you are still running single test here.
the loop should be in the context and not inside the same test case, if something will fail it will be very hard to debug

Reply:
The original test was in the same `It` (I confused it with subsystem test when I said otherwise).
I can separate it tough.

Reply:
you should

Reply:
Done.